libs/tokens/src/pure/TokenLogo/SingleLetterLogo.tsx

```diff
@@ -0,0 +1,20 @@
+import { UI } from "@cowprotocol/ui"
+import styled from "styled-components/macro"
```

Comment:
Not a big deal, but double quotes here :)

Reply:
Damn vscode.
Still not working quite right yet.
Will fix it.